src/components/form/signIn/index.tsx

```diff
@@ -13,6 +14,8 @@ import {
 
 const SignInForm = () => {
   const { handleSubmit, register, errors, onSignInSubmit } = useAuthForm();
+  const { t } = useTranslation();
```

Comment:
In general you can return `t` from `useFormHelpers` and don't use an additional `useTranslation()`

Reply:
fix: useTranslation removed. Now useFormHelpers accepts 't' as a prop =>
const useFormHelpers   = (t: (key: string) => string) 
Exp: const { SingUpFormHelper } = useFormHelpers(t);

Reply:
You could make a vice versa. const { t, SignInFormHelper } = useFormHelpers;
But ok, thats fine